vcpkg.json

```diff
@@ -2,6 +2,10 @@
   "name": "ziti",
   "version-semver": "1.0.0",
   "dependencies": [
+    {
```

Comment:
This will cause vcpkg to download and build libacl for all linux builds. Would it be better to use/depend on the distro-provided packages where available? If so, we have a pattern for using distro dependencies in vcpkg-overlays/linux-syslibs.

Reply:
Do we have a way to specify to `vcpkg` that it should conditionally build a dependency instead?  This port made working with `libacl` nice since the pkg-config file `libacl.pc` results in a build where `libacl.a` is statically linked.